xml/planning-architecture-alternative-rhel_compute_model.xml

```diff
@@ -6,6 +6,153 @@
   xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>&rhla; Compute Nodes</title>
+ <para>
+  There are several tasks you need to perform in order to install the software
+  required for &rhel; &compnode;s. Prior to this, you need to configure &obs;
+  &centos; software repository on the deployer node and on the &rhel; &compnode;s
+  using the following URL:
+  <link xlink:href="https://download.opensuse.org/repositories/systemsmanagement:/Ardana:/8:/Cent OS/CentOS_7.3">https://download.opensuse.org/repositories/systemsmanagement:/Ardana:/8:/Cent OS/CentOS_7.3</link>
```

Comment:
Don't repeat the URL, just make this a self-closing tag -- that prevents errors when this URL needs to be updated.

Also, I think you need to remove the " " in CentOS from the URL.